pkg/sentry/syscalls/linux/vfs2/vfs2.go

```diff
@@ -163,6 +163,7 @@ func Override() {
 
 	// Override ARM64.
 	s = linux.ARM64
+	s.Table[2] = syscalls.Supported("io_submit", IoSubmit)
```

Comment:
should it be PartiallySupported?

Reply:
> should it be PartiallySupported?

Yes, right, fadvise64 and fallocate should be also PartiallySupported, I will fix it.